tests/integration/pilot/testdata/cert.pem

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
We have a bunch of certs in a certain layout under tests/testdata/certs - in particular a root CA, cert for pilot, etc 

Would be nice if all tests used the same certs - so they can reuse stuff. The certs are also quite nice when running locally.